src/models/heroModel.js

```diff
@@ -1,20 +1,33 @@
 class Hero {
-  constructor(id, name, userId) {
+  constructor(
+    id,
+    name,
+    userId,
+    experience = 0,
+    level = 1,
+    healthmax = 500,
+    healthact = 500,
+    attackmin = 1,
+    attackmax = 5,
+    defense = 1,
+    inventory = [],
+    finalWords = 'Feck! Arse! Drink!') {
```

Comment:
finalWords should be null by default

Reply:
the constructor should take an object instead of n params. this is against clean code. if you remove the 7th param, you rewrite the function.

Reply:
Oh my this was dumb. Fixed.